packages/util/src/error-with-cause.ts

```diff
@@ -15,6 +20,11 @@ export class ErrorWithCause extends Error {
     Object.setPrototypeOf(this, new.target.prototype); // restore prototype chain
     this.name = 'ErrorWithCause';
 
+    // Axios removed the stack property in version 0.27 which gave no useful information anyway. This adds the http cause.
+    if (isAxiosError(cause)) {
+      this.stack = `${this.stack}${unixEOL}Caused by:${unixEOL}Http Response: ${cause.message} - ${cause?.response?.data}`;
```

Comment:
- `cause` is an `Error`, so you don't have to check `cause?`?
- when `cause?.response` is `undefined` the `stack` will be `... Http Response: message - `

Reply:
- what do you mean by "cause is an Error, so you don't have to check cause??"?
-   on an axios error the response property is there